platforms/nuttx/CMakeLists.txt

```diff
@@ -241,8 +257,71 @@ if (NOT CONFIG_BUILD_FLAT)
 
 		DEPENDS px4 px4_kernel
 	)
-
 else()
+## TODO REMOVE ME
+target_link_libraries(px4 PRIVATE
+
+		-nostartfiles
+		-nodefaultlibs
+		-nostdlib
+		-nostdinc++
+
+		-fno-exceptions
+		-fno-rtti
+
+		-Wl,--script=${LDSCRIPT}
+		-Wl,-Map=${PX4_CONFIG}.map
+		-Wl,--warn-common
+		-Wl,--gc-sections
+
+
+		m
+		gcc
+		)
+
+	set(nuttx_userspace)
+
+	list(APPEND nuttx_userspace
+		#drivers_userspace # init_entrypt.c
```

Comment:
Is this waiting for the next step, i.e. temporarily commented out?

Reply:
Yes need to modify this list. Building nsh does not require this lib, but it does not matter if it is still in this list.